src/components/MenuButton/MenuButton.js

```diff
@@ -0,0 +1,26 @@
+import { html } from '../../shared';
+import styles from './MenuButton.module.css';
+
+/**
+ * @param {{ text: string, onClick: (e: MouseEvent) => void, variant?: 'normal' | 'outlined', disabled?: boolean }} props
+ * @returns {HTMLButtonElement}
+ */
+function MenuButton({ text, onClick, variant, disabled }) {
```

Comment:
Tutaj napisałbym variant = 'normal', skoro to ma być domyślna wartość. Później to sprawdzasz i ustawiasz normal korzystając z operatora ??
Zwiększy to czytelność kodu i jak będziesz musiał wykorzystać wartość jeszcze kilka razy to nie będziesz musiał tego znowu sprawdzać